ipwb/check_for_update.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import re
+
+import requests
+
+from ipwb import __version__ as ipwb_version
+
+logger = logging.getLogger(__name__)
+
+
+def get_latest_ipwb_version() -> str:
+    """Fetch latest IPWB version from PyPI."""
+    response = requests.get('https://pypi.python.org/pypi/ipwb/json').json()
+    return response['info']['version']
+
+
+def check_pypi_for_update():
+    """Check if IPWB is outdated and print out a message if it is."""
+    latest = get_latest_ipwb_version()
+    current = re.sub(r'\.0+', '.', ipwb_version)
+
+    if current != latest and current is not None:
```

Comment:
There is a higher chance of `latest` being `None` than `current` which is a local information and should be accessible if it is imported properly. I think the `None` check was performed due to the complications of relative imports in Py3 when a file is executed as a script in different scenarios. Also, I think check for `None` should be placed prior to any equality/inequality checks in general.

Reply:
This is a piece of original code, but the point is valid.